deps/chakrashim/chakracore.gyp

```diff
@@ -7,11 +7,10 @@
     'msvs_windows_target_platform_version_prop': '',
     'icu_args%': '',
     'icu_include_path%': '',
+    'icu_lib_path%': '',
     'linker_start_group%': '',
     'linker_end_group%': '',
     'chakra_libs_absolute%': '',
-    'icu_lib_path%': '',
-    'chakra_build_flags': '',
 
     # xplat (non-win32) only
     'chakra_config': 'Release',     # Debug, Release
```

Comment:
@jianchun are test builds of ChakraCore built with -g these days? I had a change locally that did that but I don't remember if I ever merged it...I don't think release builds are built with -g right now.